gnucash/gnome-utils/gnc-file.c

```diff
@@ -881,10 +881,9 @@ gnc_post_file_open (GtkWindow *parent, const char * filename, gboolean is_readon
             break;
         default:
             /* Can't use the given file, so just create a new
-             * database so that the user will get a window that
-             * they can click "Exit" on.
+             * Gnucash window so they can choose File->New, File->Open
+             * or just "Exit" on.
```

Comment:
The "on" at the end doesn't make sense.

Reply:
Thanks, missed that on the change of note.